src/main/java/com/wimbli/WorldBorder/Config.java

```diff
@@ -341,6 +342,17 @@ public static boolean DynmapBorderEnabled() {
         return dynmapEnable;
     }
 
+    public static void setDynmapLayerLabel(String label) {
+        dynmapLayerLabel = label;
+        log("DynMap layer label is now set to: " + label);
+        save(true);
+        DynMapFeatures.showAllBorders();
+    }
+
+    public static String DynmapLayerLabel() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
You're totally right and it kinda hurts to do it like this. Due to the fact that the entire file follows this odd "convention", that the getter does not have a "get" prepended and starts uppercase, I've also did this.

Reply:
noticed. It is code-style advice.